src/AzureServerless/Properties.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<Content xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Object id="AZURE_SERVERLESS">
+    <Group>
+      <Id>Cloud</Id>
+      <Name>Cloud Settings</Name>
+      <Description>Settings for Azure serverless deployment.</Description>
+      <Type>Main</Type>
+      <Childs>
+	      <Prop>
+          <Id>AZURE_SERVERLESS_FUNCTION_APP</Id>
+          <Name>Function app</Name>
+          <Description>Name of the Function app in Azure. It must start with a letter and only contain letters, numbers, _, and -, up to 127 characters in length.</Description>
+          <Type>Text</Type>
+          <Options>
+            <Option type='Required'/>
+          </Options>
+        </Prop>
+	      <Prop>
+          <Id>AZURE_SERVERLESS_RESOURCE_GROUP</Id>
+          <Name>Resource group</Name>
+          <Description>Name of the resource group where the function app is defined in Azure.</Description>
+          <Type>Text</Type>
+		      <Options>
+            <Option type='Required'/>
+          </Options>
+        </Prop>
+        <Group>
+          <Id>ServiceIdentityConfiguration</Id>
+          <Name>Identity Configuration (Service Principal)</Name>
+          <Description>An Azure Service Principal is an identity created for use with applications, hosted services, and automated tools to access Azure resources.</Description>
+          <Type>Main</Type>
+          <Childs>
+          <Prop>
+            <Id>AZURE_SERVERLESS_SP_APP_ID</Id>
+            <Name>Application Id</Name>
+            <Description>Application (client) ID of the Service Principal.</Description>
+            <Type>Text</Type>
+            <Options>
+            <Option type='Required'/>
+            </Options>
+          </Prop>
+          <Prop>
+            <Id>AZURE_SERVERLESS_SP_TENANT_ID</Id>
+            <Name>Tenant Id</Name>
+            <Description>Directory (tenant) ID of the Service Principal.</Description>
+            <Type>Text</Type>
+            <Options>
+              <Option type='Required'/>
+            </Options>
+          </Prop>
+          <Prop>
+              <Id>AZURE_SERVERLESS_SP_CREDENTIALS</Id>
+            <Name>Credentials</Name>
+            <Description>Password of the Service Principal.</Description>
+            <Type>Text</Type>
+            <Options>
+              <Option type='Password'/>
+            </Options>
+          </Prop>
+          </Childs>
+        </Group>	
+	    </Childs>
+    </Group>
+    <Group>
+      <Id>AzServerlessSettings</Id>
+      <Name>Serverless Deploy Settings</Name>
+      <Description>Serverless deploy settings</Description>
+      <Type>Main</Type>          
+        <Childs>
+          <Prop>
+            <Id>AZURE_SERVERLESS_OPENAPI_FILE</Id>
```

Comment:
What do we expect in this Property? We need to specify an open api file? Which one?

I thought that we would generate it automatically.

Reply:
There are two deployment targets
1. For Azure functions of different types (queue, http..) In this case the yaml is generated by the deploy engine, as you said.
2. This target (the one of the PR), is for the services of an app. In this case, I think it's better to ask the user to activate the Generate Open API property for all the objects of the deployment unit, specify, generate, and get de yaml file to pass it to the deploy. 
I wrote this in the spec: https://iwiki.genexus.com/wiki.aspx?29416,Spec+Azure+functions+de+tipo+Http#Dependencias
We can check it if you want.